src/core/auth/authentication/components/KratosUI.tsx

```
@@ -45,11 +45,57 @@ const toAlertVariant = (type: string) => {
+
+    const renderLineWithHtml = (content: string) => {
+      // Handle HTML tags like <strong>, <em>, <i> by using dangerouslySetInnerHTML
+      if (
+        content.includes('<strong>') ||
+        content.includes('</strong>') ||
+        content.includes('<em>') ||
+        content.includes('</em>') ||
+        content.includes('<i>') ||
+        content.includes('</i>')
+      ) {
+        return <span dangerouslySetInnerHTML={{ __html: content }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)